projects/uitdatabank/models/ownership.json

```diff
@@ -7,20 +7,21 @@
       "$ref": "./common-string-uuid.json"
     },
     "itemId": {
-      "$ref": "./common-string-uuid.json"
+      "type": "string",
```

Comment:
This is the same as the one from `ownership-post.json`, maybe create a follow-up pull request to create a `common-legacy-uuid`?